### DISCUSSION TITLE
idAccessor on rowExpansion?

### DISCUSSION THREAD

**Discussion by Zaqttack** [icflorescu/mantine-datatable#117] [Q&A]
After following the guide on how to implement rowExpansion, I was able to click on one row but it was opening all of the rows. I realized that this was because I needed to signify a key/id to each entry with my .json file. Though, not until I discovered issue #90 was I able to find that I needed to ad `idAccessor` to my Datatable component. But this lead me to possibly submit an issue to update the docs but I then discovered that even your examples do not use `idAccessor`. 

So, my question is, am I potentially implementing my records incorrectly, or should `idAccessor="name"` be added to the examples on the [docs](https://icflorescu.github.io/mantine-datatable/examples/expanding-rows) page?

**Comment by icflorescu** [comment]
The `idAccessor` property defaults to `id`, so you only need to define it if your data has a different unique key (such as `name`).
This is covered in the docs here: https://icflorescu.github.io/mantine-datatable/examples/non-standard-record-ids.
That being said, maybe I should add a note on using non-standard IDs on the row expansion example page too.

**Comment by icflorescu** [comment]
Note added on the docs page.
